dist/js/stacks.js

```diff
@@ -1997,6 +1997,32 @@ Copyright © 2019 Basecamp, LLC
 })();
 
 
+;
+
+(function() {
+    "use strict";
+
+    Stacks.addController("s-modal", {
+        targets: ["content"],
+
+        show: function() {
+            this.contentTarget.setAttribute("tabindex", 0);
+            this.contentTarget.setAttribute("aria-hidden", false);
+            document.body.setAttribute("tabindex", -1);
+            document.body.setAttribute("aria-hidden", true);
+        },
+        hide: function() {
```

Comment:
How difficult is wiring the `esc` key to this `hide` function?

Reply:
Can do

Reply:
703a726